examples/resnet-18/resnet-18.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+from transformers import AutoFeatureExtractor, AutoModelForImageClassification
+import torch
+from shark_turbine.aot import *
+from iree.compiler.ir import Context
+from iree.compiler.api import Session
+import iree.runtime as rt
+from datasets import load_dataset
+
+# Loading feature extractor and pretrained model from huggingface
+extractor = AutoFeatureExtractor.from_pretrained("microsoft/resnet-18")
+model = AutoModelForImageClassification.from_pretrained("microsoft/resnet-18")
+
+# load an example
+dataset = load_dataset("huggingface/cats-image")
+image = dataset["test"]["image"][0]
+
+# if you want to see the cat picture:
+# image.save("cats-image.jpg")
+
+# if you want to run a custom image through inference.
+# import PIL
+# image = PIL.JpegImagePlugin.JpegImageFile("yourexamplepicture.jpg")
+
+# extract features from image to feed to model
+inputs = extractor(image, return_tensors="pt")
+pixel_tensor = inputs.pixel_values
+
+
+# define a function to do inference
+# this will get passed to the compiled module as a jittable function
+def forward(pixel_values_tensor: torch.Tensor):
+    with torch.no_grad():
+        logits = model.forward(pixel_values_tensor).logits
+    predicted_id = torch.argmax(logits, -1)
+    return predicted_id
+
+
+# A dynamic module for doing inference
+class RN18(CompiledModule):
```

Comment:
minor: worth mentioning this is an aot example somewhere

Reply:
Okay, I mentioned that in a few places.